pluginconfig/commit.go

```diff
@@ -9,58 +9,23 @@ import (
 	"strings"
 	"time"
 
-	"github.com/smartcontractkit/chainlink-common/pkg/merklemulti"
 	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
 
+	"github.com/smartcontractkit/chainlink-common/pkg/merklemulti"
+
 	commonconfig "github.com/smartcontractkit/chainlink-common/pkg/config"
 	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
 )
 
 // EvmDefaultMaxMerkleTreeSize is the default number of max new messages to put in a merkle tree.
 // We use this default value when the config is not set for a specific chain.
```

Comment:
Lets move this comment closer to the var declaration

Reply:
I don't think this comment is adding much meaning to the var name. I'll just remove it.